finger_torque: -0.6
